spec/requests/mentors/destroy_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe MentorsController, type: :request do
+  describe 'destroy' do
+    context 'when destroy a mentor' do
+
+      let!(:mentor_one) { Mentor.create(name:'Vitor', email:'vitor@teste.com', bio: 'oi, sou o vitor e tenho 22 anos') }
+      
+      it 'returns status 302' do
```

Comment:
Sempre que vc precisa usar `let!` é um smell de que algo não está certo.
Nesse caso, como vc só tem um cenário no contexto, o indicado é criar a variável diretamente dentro do `it`

```suggestion
      it 'returns status 302' do
      
      mentor_one = Mentor.create(name:'Vitor', email:'vitor@teste.com', bio: 'oi, sou o vitor e tenho 22 anos')
```